README.md

```diff
@@ -61,11 +72,56 @@ To authenticate your identity, we use a personal access token that we call `regc
             - name: regcred
 ```
 
-The code snippet above is from the cronjob.yml in `k8s-deploy-bke`. An image pull secret is used to authenticate a Kubeneretes cluster to pull images from a container registry, in our case: `ghcr.io`. The`regcred` secret is a Personal Access Token generated from Github that has `read`, `write`, and `delete` powers, currently generated by Josh Lu (@jashlu). 
+The code snippet above is from each cronjob.yml file in `k8s-deploy-bke`. An image pull secret is used to authenticate a Kubeneretes cluster to pull images from a container registry, in our case: `ghcr.io`. This `regcred` secret is a [fine-grained Personal Access Token](https://github.com/settings/tokens?type=beta) generated by the CCV CJ bot that has `read` access for **content** and **metadata**, and repository access for all project portal related content repositories and the projectportal-email-service repository.
 
-The following command was used to create the regcred secret 
+Use the following command to create the regcred secret, using the fine-grained personal accesss token you generated previously.
 
 ```
 kubectl create secret docker-registry regcred --docker-server=ghcr.io --docker-username=USERNAME --docker-password=PERSONAL_ACCESS_TOKEN
 ```
-`kubectl create secret docker-registry` specifies the type of secret we want, which in this case, is of type `docker-registry`. This is because the credentials are used to authenticate with the Docker registry. This command will store the newly created `regcred` secret in the current context's namespace, in this case, `tpl-projectportal`. The secret is base64-encoded, ensuring no sensitive info is stored in plain text.  
+
+`kubectl create secret docker-registry` specifies the type of secret we want, which in this case, is of type `docker-registry`. This is because the credentials are used to authenticate with the Docker registry. This command will store the newly created `regcred` secret in the current context's namespace, in this case, `tpl-projectportal`. The secret is base64-encoded, ensuring no sensitive info is stored in plain text.
+
+###### Maintaining Regcred Secret
+The Personal Access Token is set up to expire after a specific period of time, requiring manual renewal by CCV personnel with access to the CCV CJ Bot. 
```

Comment:
When this PAT is regenerated, the email service needs to be re-deployed (see "Deploying the email-service")